src/my_assistant.py

```diff
@@ -33,8 +41,11 @@
 
 # Presa in input una stringa, questa verrà esposta a voce
 def speak(response: str) -> None:
-    engine.say(response)
-    engine.runAndWait()
+    if sys.platform == 'Linux':
+        engine.say(response)
+        engine.runAndWait()
+    elif sys.platform == 'darwin':
+        os.system(f"echo '{response}' | espeak -v it")
```

Comment:
Ti ringrazio per il feedback, avevo solo un appunto da fare, in questo modo utilizzando windows le risposte non verranno date a voce manca infatti il caso in cui `sys.platform == 'win32'`.
Quindi potresti scrivere:
```suggestion
    if sys.platform == 'Linux' or sys.platform == 'win32':
        engine.say(response)
        engine.runAndWait()
    elif sys.platform == 'darwin':
        os.system(f"echo '{response}' | espeak -v it")
```

Reply:
fatto ✅